### DISCUSSION TITLE
difference of quiz and exam?

### DISCUSSION THREAD

**Discussion by KlavsKlavsen** [inducer/relate#917] [Q&A] [answered]
I have both quiz'es and exams I can create - but it is unclear what the actual differences are?

**Comment by inducer** [comment, marked as answer]
To Relate, there are only "flows". Those are mainly differentiated by their "flow rules", see examples here:

https://documen.tician.de/relate/flow.html#sample-rule-sets

So, what do you mean by "quiz" and "exam"?
